src/styles/variables.scss

```
@@ -0,0 +1,16 @@
+$calendar-size: 100px;
+$calendar-gap: $calendar-size / 100;
+$calendar-border: $calendar-size / 100;
+$calendar-padding: $calendar-size / 10;
+$calendar-font-size: 0.3 * $calendar-size;
+$calendar-animation-size: 0.2 * $calendar-size;
+$calendar-flex-width: (
+  (7 * $calendar-size) + ($calendar-gap * 6) + ($calendar-padding * 2)
+);
+$calendar-flex-height: ((4 * $calendar-size) + (3 * $calendar-gap));
+$week-days: '-mon', '-tue', '-wed', '-thu', '-fri', '-sat', '-sun';
+$counter: 0;
```

instead of counter you can add values next to all week-days

```suggestion
$week-days: 'mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6;
```